### ISSUE TITLE
Log Lighthouse results on merge-to-latest

### ISSUE THREAD

**Issue by ChrisBAshton** [bbc/simorgh#1196]
**Is your feature request related to a problem? Please describe.**
We want to persist Lighthouse results so we can monitor performance over time.

Dependent on #1183 as sometimes Lighthouse returns null, which wouldn't be very useful. But we _could_ get started on this before making Lighthouse less flaky.

Also depends on us adding a Lighthouse / Performance section to Grafana. (But this could be done _first_).

**Describe the solution you'd like**
When a branch is merged into latest, we want to retrieve the lighthouse score and push it to S3 logging so that it can be surfaced on Grafana.

**Describe alternatives you've considered**
Not doing this, and having to roughly remember our performance level over time.

**Testing notes**
[Tester to complete]

Dev insight: When a PR is merged to latest, see the Grafana dashboard update with latest Lighthouse scores.

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)
